model/schema/biologicalEntitySet.yaml

```diff
@@ -56,18 +56,32 @@ classes:
     slots:
       - genes
 
+  GeneClass:
+    description: >-
+      A gene class is a set of genes which is of biological significance, and
```

Comment:
Is it possible to describe this in more detail - what does biological significance mean?  And can you give an example of the kinds of phenotype attributed to a group of genes?

Reply:
description improved